libindy_vdr/src/pool/requests/prepared_request.rs

```diff
@@ -80,6 +81,18 @@ impl PreparedRequest {
         }
     }
 
+    pub fn get_cache_key(&self) -> VdrResult<String> {
+        let mut req_json = self.req_json.clone();
+        let req_map = req_json
+            .as_object_mut()
+            .ok_or_else(|| input_err("Invalid request JSON"))?;
+        req_map.remove("reqId");
+        req_map.remove("signature");
```

Comment:
This still refers to the `req_json`, right? Otherwise the removes don't do anything as `req_json` is being used after.

Reply:
yes req_map is a mutable reference to req_json, but in a slightly different type. `as_object_mut()` returns a mutable reference to req_json in the form of a `&mut Map<String, Value>` object. Insertions and removals made to the map are still reflected in the original object. I think what's happening here is that `as_object_mut()` is returning a reference to the underlying struct that serde_json uses in it's `Value` type

Reply:
I've tested this and can confirm that the `reqId` and `signiture` attributes are removed before making the raw key